clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/unsecured/OAuthBearerUnsecuredValidatorCallbackHandler.java

```diff
@@ -170,33 +170,31 @@ private void handleCallback(OAuthBearerValidatorCallback callback) {
                 .throwExceptionIfFailed();
         OAuthBearerValidationUtils.validateTimeConsistency(unsecuredJwt).throwExceptionIfFailed();
         OAuthBearerValidationUtils.validateScope(unsecuredJwt, requiredScope).throwExceptionIfFailed();
-        log.info("Successfully validated token with principal {}: {}", unsecuredJwt.principalName(),
-                unsecuredJwt.claims().toString());
+        if (log.isInfoEnabled()) {
```

Comment:
Is this condition check necessary? It is rare in code base.

Reply:
OK, I think you are right.  We do not need condition check in here.  I already revert it.